.deepsource.toml

```diff
@@ -0,0 +1,19 @@
+version = 1
+
+test_patterns = [
+    "tests/**",
+    "**/test_*.py"
+]
+
+exclude_patterns = [
+    "examples/**",
```

Comment:
We can also include `examples` which contains nothing but python code.  Even though not part of release package, I think we can still include them within code quality check.  Wdyt?

Reply:
Sounds right, I'll remove `examples/` from the excluded patterns.